scripts/zones/Bastok_Markets/npcs/Linkshell_Concierge.lua

```diff
@@ -0,0 +1,58 @@
+-----------------------------------
+-- Area: Bastok Markets
+-- NPC: Linkshell_Concierge
+-----------------------------------
+package.loaded["scripts/zones/Bastok_Markets/TextIDs"] = nil;
+-----------------------------------
+require("scripts/zones/Bastok_Markets/TextIDs");
+require("scripts/globals/concierge");
+require("scripts/globals/settings");
+-----------------------------------
+
+function onTrade(player,npc,trade)
+end;
+
+function onTrigger(player,npc)
+
+    if (npc:getID() == 17739958) then -- the one on the right
```

Comment:
instead of using NPC ID's ins cript link this, its preferred to either:
A) rename the npc's to tell them apart in the sql and then use a script for each. This method future proofs the script against client updates changing the ID and we just have to update the sql.
-OR-
B) put the IDs in the zones mobIDs.lua require that in teh npc script and call teh variable here instead of the ID. That keeps all the IDs in one file so we can more easily verify/update them when they shift (they move nearly every client update, its very annoying). This method requires us to keep on top of ID updates in both script *and* the sql but avoids extra scripts.

Reply:
Which one would you prefer / is most commonly used in DSP ?
Editing the sql some more, and adding files, or adding to mobids ?

Reply:
I think its worth doing (A) but will merge either one